.travis.yml

```diff
@@ -19,6 +19,8 @@ cache:
 before_install:
   - export NG_CLI_ANALYTICS=ci
   - export GO111MODULE=on
+  - echo $GOPROXY
+  - export $GOPROXY=https://proxy.golang.org
```

Comment:
Remove the `$` when exporting a variable.

Also all examples I've seen setting GOPROXY don't include the protocol (see https://twitter.com/JetstackHQ/status/1158698823910395905 for example)

Reply:
I found following log in travis job.
```
Setting environment variables from repository settings
$ export DOCKER_CLI_EXPERIMENTAL=enabled
$ export GOPROXY=proxy.golang.org
```
That may the reason to fail.

Reply:
I added "http://", jobs seem to run successfully.

Reply:
Failures in other PR are actually as follow:
```
invalid $GOPROXY setting: malformed URL or invalid scheme (must be http, https, file)
```

Schema would be needed, I think.  😉

Reply:
Updated GOPROXY to be https:// and testing against #4171

Reply:
Do changes about `repository settings` be tested? If not, it's better to migrate their settings to `.travis.yaml`, I think.

Reply:
Environment settings (currently) we manage within Travis itself (for secrets and whatnot)

This kind of change would be nice to be something we could keep in the `.travis.yml` file BUT I was in the middle of testing the GOPROXY variable change (in Travis) almost at the same time you were submitting the PR. (Great minds think alike!)

I personally think (for now) we should keep the change in Travis (so we can close this PR) and (soon) migrate any non-secret config info out of Travis (itself) and into .travis.yml

Thoughts @shu-mutou ?

Reply:
I updated this PR into migration of environment variables for travis. 
We can consider in other PR or issue whether we should set GOPROXY or not. :wink: